apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V53__preadj_mbi_caching-1.sql

```diff
@@ -0,0 +1,37 @@
+/*
+ * Table to hold known MBI and their hashed equivalents.
+ */
+CREATE TABLE "pre_adj"."MbiCache" (
+    "mbiId"   bigint      NOT NULL PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    "mbi"     VARCHAR(13) NOT NULL,
+    "hash"    VARCHAR(64) NOT NULL,
+    "oldHash" VARCHAR(64),
+    "lastUpdated" timestamp with time zone
+);
+
+/*
+ * We need these indexes to support queries by mbi, hash, and oldHash.  Note that oldHash
+ * is usually null (only set to a value during hash rotation) so the index cannot be unique.
+ */
+CREATE UNIQUE INDEX "MbiCache_mbi_idx" on "pre_adj"."MbiCache"("mbi");
+CREATE UNIQUE INDEX "MbiCache_hash_idx" on "pre_adj"."MbiCache"("hash");
+CREATE INDEX "MbiCache_oldHash_idx" on "pre_adj"."MbiCache"("oldHash");
+
+/*
+ * Add foreign key to reference MbiCache in claims tables.
+ */
+ALTER TABLE "pre_adj"."FissClaims"
```

Comment:
We'll need to also disable hibernate validation before we mege this PR

Reply:
Since we are only adding a column hibernate validation won't care.  It only cares about missing columns or data type changes in the columns that it knows about.